### ISSUE TITLE
Let nuget-prepare command download the nuget packages as well

### ISSUE THREAD

**Issue by genlu** [dotnet/roslyn-tools#1205]
Right now we still need to manually figure out which roslyn version is inserted into a given VS build, and go download the right packages first. Maybe we can let the tool handle this part as well

**Comment by davidwengier** [comment]
Yes! If you run the `vsbranchinfo` command it will tell you which versions are used, and offer links to download, so the code for this is all there in bits and pieces.

Combining these things together definitely makes sense!

**Comment by genlu** [comment]
Oh boy, I wasn't even aware of `vsbranchinfo`, that's awesome! Although I just ran into an error

```
> roslyn-tools vsbranchinfo
Roslyn info from VS branch main:
  Package Version: 4.3.0-1.22218.8
Error occurred: Couldn't find build for build number: 20220418.8
```
